app/controllers/burgers_controller.rb

```diff
@@ -4,4 +4,29 @@ def index
     @burgers = Burger.all
   end
 
+  def show
+    @burger = Burger.find(params[:id])
+  end
+
+  def new
+    @burger = Burger.new
+  end
+
+  def create
+    @burger = Burger.new(burger_params)
+    @current_user = current_user
+    @burger.user_id = current_user.id
```

Comment:
You could also do:
`@burger.user_id = current_user.id`
or even
`@burger.user = current_user` (of this I'm not sure but I think it works)